src/System.Numerics.Vectors/tests/QuaternionTests.cs

```diff
@@ -383,7 +383,7 @@ public void QuaternionCreateFromYawPitchRollTest2()
 
                         Quaternion expected = yaw * pitch * roll;
                         Quaternion actual = Quaternion.CreateFromYawPitchRoll(yawRad, pitchRad, rollRad);
-                        Assert.True(MathHelper.Equal(expected, actual), String.Format("Yaw:{0} Pitch:{1} Roll:{2}", yawAngle, pitchAngle, rollAngle));
+                        Assert.True(MathHelper.Equal(expected, actual), $"Quaternion.QuaternionCreateFromYawPitchRollTest2 Yaw:{yawAngle} Pitch:{pitchAngle} Roll:{rollAngle} did not return the expected value: expected {expected} actual2 {actual}");
```

Comment:
Any reason this says `actual2 {actual}`?  Is that just a type-o?

Reply:
type-o

Reply:
Fixed